<launch>

    <arg name="isSim" default="true" />
    <arg name="config_file" default="pathTracker_sim1" />

    <!-- 0 -> nav_msgs::Odometry -->
    <!-- 1 -> geometry_msgs::PoseWithCovarianceStamped -->
    <!-- Remember to change the topic name. -->
    <arg name="odom_type" value="0" if="$(arg isSim)" />
    <arg name="odom_type" value="1" unless="$(arg isSim)" />

    <node name="path_tracker" pkg="path_tracker" type="pathTracker" output="screen">

        <remap from="cmd_vel" to="raw_cmd_vel" />
        <remap from="nav_goal" to="path_tracker_goal" />

        <rosparam file="$(find path_tracker)/params/$(arg config_file).yaml" command="load" />

        <param name="odom_type" type="int" value="$(arg odom_type)" />
        <param name="odom_topic_name" type="string" value="odom" if="$(arg isSim)" />
        <param name="odom_topic_name" type="string" value="ekf_pose" unless="$(arg isSim)" />

    </node>

</launch>